addons/mail/static/src/core/web/recipients_input_tags_list_popover.js

```diff
@@ -28,8 +28,11 @@ export class RecipientsInputTagsListPopover extends Component {
         });
     }
 
+    /**
+     * @deprecated
+     */
```

Comment:
For posterity, can you tell me why the method is deprecated?

Reply:
Because the `id` getter is never used in the standard Odoo codebase :sweat_smile: